<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="63148" started-at="2018-01-30T20:35:48Z" finished-at="2018-01-30T20:36:51Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="63148" started-at="2018-01-30T20:35:48Z" finished-at="2018-01-30T20:36:51Z">
      <class name="test.cuenta.bancaria.CrearCuenta">
        <test-method status="PASS" signature="initializeTestBaseSetup()[pri:0, instance:test.cuenta.bancaria.CrearCuenta@59fa1d9b]" name="initializeTestBaseSetup" is-config="true" duration-ms="7245" started-at="2018-01-30T15:35:48Z" finished-at="2018-01-30T15:35:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeTestBaseSetup -->
        <test-method status="FAIL" signature="crearCuenta()[pri:0, instance:test.cuenta.bancaria.CrearCuenta@59fa1d9b]" name="crearCuenta" duration-ms="55454" started-at="2018-01-30T15:35:55Z" finished-at="2018-01-30T15:36:51Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Unable to locate element: //html/body/div[2]/div[2]/div/div[3]/fieldset/div/div/table/tbody/tr/td[2]
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.6.0', revision: '6fbf3ec767', time: '2017-09-27T16:15:26.402Z'
System info: host: 'CAJA200', ip: '192.168.56.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_151'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{moz:profile=C:\Users\WINDOW~1\AppData\Local\Temp\rust_mozprofile.ZJiAG804OtdG, rotatable=false, timeouts={implicit=0, pageLoad=300000, script=30000}, pageLoadStrategy=normal, platform=XP, specificationLevel=0, moz:accessibilityChecks=false, acceptInsecureCerts=true, browserVersion=54.0, platformVersion=10.0, moz:processID=8076, browserName=firefox, javascriptEnabled=true, platformName=XP}]
Session ID: 91faa46e-a0ec-4805-91b9-e378cd298ffb
*** Element info: {Using=xpath, value=//html/body/div[2]/div[2]/div/div[3]/fieldset/div/div/table/tbody/tr/td[2]}]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Unable to locate element: //html/body/div[2]/div[2]/div/div[3]/fieldset/div/div/table/tbody/tr/td[2]
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.6.0', revision: '6fbf3ec767', time: '2017-09-27T16:15:26.402Z'
System info: host: 'CAJA200', ip: '192.168.56.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_151'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{moz:profile=C:\Users\WINDOW~1\AppData\Local\Temp\rust_mozprofile.ZJiAG804OtdG, rotatable=false, timeouts={implicit=0, pageLoad=300000, script=30000}, pageLoadStrategy=normal, platform=XP, specificationLevel=0, moz:accessibilityChecks=false, acceptInsecureCerts=true, browserVersion=54.0, platformVersion=10.0, moz:processID=8076, browserName=firefox, javascriptEnabled=true, platformName=XP}]
Session ID: 91faa46e-a0ec-4805-91b9-e378cd298ffb
*** Element info: {Using=xpath, value=//html/body/div[2]/div[2]/div/div[3]/fieldset/div/div/table/tbody/tr/td[2]}
	at org.testng.Assert.fail(Assert.java:93)
	at test.cuenta.bancaria.CrearCuenta.crearCuenta(CrearCuenta.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- crearCuenta -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.cuenta.bancaria.CrearCuenta@59fa1d9b]" name="tearDown" is-config="true" duration-ms="424" started-at="2018-01-30T15:36:51Z" finished-at="2018-01-30T15:36:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- test.cuenta.bancaria.CrearCuenta -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
